Develop

Archi1993 opened the pull request.

Archi1993 committed:
add gitignore

Archi1993 committed:
add pom
add src

egorvas commented on src/test/java/praktikum/CheckScooterOrderTest.java:
```diff
@@ -0,0 +1,81 @@
+    public void testOrdersStartsToClickOrderTopButton() {
+        WebDriver driver = driverRule.getDriver();
+        MainPage objMainPage = new MainPage(driver);
+        objMainPage.clickOrderTopButton();
```
**Можно лучше:** у тебя фактически два одинаковых теста, можешь вынести кнопку заказа в параметр и тогда можно оставить один

egorvas commented on src/test/java/praktikum/CheckTextInMainPageAccordionTest.java:
```diff
@@ -0,0 +1,61 @@
+
+
+    @Test
+    public void CheckTextInAccordion() {
```
**Нужно исправить:** функции java всегда с маленькой буквы

egorvas commented on src/test/java/praktikum/CheckTextInMainPageAccordionTest.java:
```diff
@@ -0,0 +1,61 @@
+        WebDriver driver = driverRule.getDriver();
+        MainPage objMainPage = new MainPage(driver);
+        objMainPage.scrollToMainQuestionLabel(driver);
+        driver.findElement(By.xpath(String.format("%s%d']", objMainPage.getAccordionLocator(), accordionLocator))).click();
```
**Нужно исправить:** вся работа с локаторами должна быть вынесена в PO

Archi1993 committed:
Исправление в коде 1: Функции java всегда с маленькой буквы
Исправление в коде 2: Вся работа с локаторами должна быть вынесена в PO, добавил новый метод buildLocatorAndClick и вызываю его в тесте

Archi1993 committed:
Исправление в коде 1: Функции java всегда с маленькой буквы
Исправление в коде 2: Вся работа с локаторами должна быть вынесена в PO, добавил новый метод buildLocatorAndClick и вызываю его в тесте
